sonar-db/src/main/java/org/sonar/db/version/v55/CreateInitialSchema.java

```diff
@@ -915,4 +915,8 @@ private void addIndex(Context context, String table, String index, boolean uniqu
     }
     context.execute(builder.build());
   }
+
+  private static VarcharColumnDef.Builder newVarcharBuilder(String column) {
```

Comment:
I think it should be clear that this method creates a "bad" varchar builder.
you could either deprecate this method or call it something like `newLenientVarcharBuilder`
at least it should be documented with some javadoc to avoid any confusion in the future